### PR TITLE
Applied changes made by Martin Kocian for the FE-I3 readout with the …

### PR DESCRIPTION
**Pull request by arummler**
…RCE (+HSIO+MCC). This also fixes the missing ToT and LVL1 histograms in the OnlineMonitor for this kind for data aquisition. Started testing in testbeam July 3rd at CERN with ATLAS telescope, needs more testing (up to now only existing FE-I4 functionality used) and potentially cleanup.

### CODE REVIEW

**Comment by simonspa** [comment]
Not merging this due to the "needs more testing and cleanup" - I'm expecting more commits from your side. If this is a misinterpretation and should be merged, let me know. It compiles - and since it's a user processor we don't care about much more. :)

**Comment by arummler** [comment]
I think the processor commit can be merged as there were no problems during data taking. If anything occures during the ongoing data analysis we can still revert or fix. I tried to run the OnlineMonitor on Windows 7 and it did not compile. Introduced two small fixes which helped. Please trigger a Travis rebuild manually as it failed to download during the first attempt, a svn hickup, and I don't want to push a dummy commit or close and reopen the PR. Now the OnlineMonitor crashes irregularly after startup (running in read from file mode) but I will open an issue for that.

eudaq_compilation_error_1
eudaq_compilation_error_2

**Comment by arummler** [comment]
The Windows thing is follow-up of issues #17 and #76.

File: monitors/onlinemon/include/SimpleStandardEvent.hh [hunk 1]
```diff
@@ -11,7 +11,7 @@ typedef char int8_t;
 #include <string>
 #include <vector>
 #include <iostream>
-#include <stdint.h>
+//#include <stdint.h>
```
Comment (simonspa) [inline monitors/onlinemon/include/SimpleStandardEvent.hh]: Is there a reason for this change?

Comment (arummler) [inline monitors/onlinemon/include/SimpleStandardEvent.hh]: It removes the nine errors in the second screen shot and it seems it is not necessary as it compiles without including stdint.h. The include files that appear in the error message are (indirectly) included by stdint.h.

**Comment by simonspa** [comment]
@arummler okay, is fine then and also compiles. TravisCI failed due to SourceForge's downtime last week and the ROOT package we pull from there.

I'm sorry that we just released eudaq 1.5.0 without the PR, but I'll merge now and it will be part of EUDAQ v1.5.1 soonish I guess. Maybe the WIndows issue is also solved until then.

**Pull request merged**